### ISSUE TITLE
[Feature Request] Multiple Domains

### ISSUE THREAD

**Issue by bds4me-thomasbrugger** [laramies/theHarvester#418]
**Feature Request or Bug or Other**
Multiple Domains from file

**Describe the feature request or bug or other**
Collection some inputs about more as one url

Maybe some trying with wget and grep make some good results:
wget --timeout=4 --waitretry=0 --tries=2 -q -O - -i url.txt | grep -E -o "\b[a-zA-Z0-9.-]+@[a-zA-Z0-9.-]+\.[de|com|at]+\b" | sort -u > emaillist.txt

**Comment by NotoriousRebel** [comment]
I don't think it would be too difficult to make it so you can pass in a file with multiple domains it would just require another for loop [here](https://github.com/laramies/theHarvester/blob/master/theHarvester/__main__.py#L158).

**Comment by leebaird** [comment]
This could be accomplished by using a Bash 'for loop.'
